packages/wonder-blocks-core/components/view.js

```diff
@@ -40,9 +51,26 @@ const StyledDiv = addStyle<"div">("div", styles.default);
  * - An `aphrodite` StyleSheet style
  * - An array combining the above
  */
-export default class View extends React.Component<TextViewSharedProps> {
+export default class View extends React.Component<Props> {
+    static defaultProps = {
+        tag: "div",
+    };
+
     render() {
-        const {testId, ...restProps} = this.props;
-        return <StyledDiv data-test-id={testId} {...restProps} />;
+        const {testId, tag, ...restProps} = this.props;
+        switch (tag) {
+            case "article":
+                return <StyledArticle data-test-id={testId} {...restProps} />;
+            case "aside":
+                return <StyledAside data-test-id={testId} {...restProps} />;
+            case "nav":
+                return <StyledNav data-test-id={testId} {...restProps} />;
+            case "section":
+                return <StyledSection data-test-id={testId} {...restProps} />;
+            case "div":
+                return <StyledDiv data-test-id={testId} {...restProps} />;
+            default:
+                return <StyledDiv data-test-id={testId} {...restProps} />;
```

Comment:
The `default` case should look something like:
```
default:
    (tag: empty);
    throw Error(`${tag} is not an allowed value for the 'tag' prop`);
```
The `(tag: empty);` line tries to case `tag` to `empty` which is never valid.  If the switch/case is exhaustive flow nows that this will never happen and does not raise an error.  Try removing one of the other cases to see what happens.

The `throw` isn't really necessary since all of our code is typed and that should never happen but since there's an expectation that `render()` return a React.Node it's better to throw to indicate that this path doesn't return a value and that's okay.  It would be cool if the tools were smart enough to know that it isn't necessary to return a value from this path, but they're not there yet.